tools/install.py

```diff
@@ -173,6 +166,7 @@ def ignore_inspector_headers(files, dest):
   action([
     'common.gypi',
     'config.gypi',
+    'config.json',
```

Comment:
Does any downstream rely on the variables being in `config.gypi` instead of being included? (Not sure who to ping but I guess this could make the PR semver-major?)

Reply:
Can't we just recreate config.gypi in this script? Or does that defeat the purpose of this PR if there is an inconsistency between what's generated in the build and what gets distributed in the tarballs?

Reply:
I have considered that. Since they are both created by `./configure` we could do that to solve this if we find it's a problem.
I wanted to do it like this to see if we can map the scope by looking for things that break.

(P.S. if we decide to make this semver-major, then I guess it would make sense to add the deprecation of `process.config` here)